php-nginx/Dockerfile

```diff
@@ -54,8 +54,9 @@ ENV NGINX_DIR=/opt/nginx \
     SESSION_SAVE_PATH=/tmp/sessions \
     NGINX_VERSION=1.10.1 \
     PHP56_VERSION=5.6.22 \
-    PHP70_VERSION=7.0.7 \
-    PATH=${PHP_DIR}/bin:$PATH
+    PHP70_VERSION=7.0.7
+
+ENV PATH=${PHP_DIR}/bin:$PATH
```

Comment:
I would prefer hard coding the path. A new instruction in the Dockerfile introduce another docker layer which will increase the image size a bit.